packages/run-protocol/src/econ-behaviors.js

```diff
@@ -442,3 +443,163 @@ export const startRewardDistributor = async ({
   );
 };
 harden(startRewardDistributor);
+
+/**
+ * @typedef { import('@endo/eventual-send/').Unpromise<T> } Unpromise<T>
```

Comment:
`Unpromise` is deprecated (in agoric-sdk before those changes were globbered). Instead use `Awaited` built into TS